pkg/driver/controller_test.go

```diff
@@ -1287,6 +1287,61 @@ func TestCreateSnapshot(t *testing.T) {
 				mockCtl.Finish()
 			},
 		},
+		{
+			name: "success: no tags",
```

Comment:
nit: I would rename it to "success: required parameters". In the future we can leverage the same test to ensure no parameters will cause errors if missing.

Reply:
I don't want to add much scope, but a test like this would be useful during CreateVolume for both a filesystem and volume.

Reply:
Ack, I'll revise the name of this unit test. To avoid merge conflicts with my other PR, I'll add the unit tests for CreateVolume in the other PR instead of this PR.